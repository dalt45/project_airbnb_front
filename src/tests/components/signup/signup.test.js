import React from 'react';
import { shallow } from'enzyme';
import SignUpForm from '../../../components/signup/form';

test('should render SearchForm page correctly', () => {
   const wrapper = shallow(<SignUpForm />);
   wrapper.find('TextField').at(0).simulate('change',{
       target:{
           value:"daniel@gmail.com"
       }
   })
   expect(wrapper.state('email')).toEqual("daniel@gmail.com")
   expect(wrapper).toMatchSnapshot();
})