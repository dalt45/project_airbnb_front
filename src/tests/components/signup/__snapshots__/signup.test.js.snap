// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SearchForm page correctly 1`] = `
<div>
  <WithStyles(Card)>
    <p>
      Registrate en Airbnb Clon
    </p>
    <TextField
      id="email"
      onChange={[Function]}
      required={false}
      select={false}
      value="daniel@gmail.com"
    />
    <TextField
      id="name"
      onChange={[Function]}
      required={false}
      select={false}
      value=""
    />
    <TextField
      id="last_name"
      onChange={[Function]}
      required={false}
      select={false}
      value=""
    />
    <TextField
      id="password"
      onChange={[Function]}
      required={false}
      select={false}
      value=""
    />
    <TextField
      id="date"
      onChange={[Function]}
      required={false}
      select={false}
      type="date"
      value=""
    />
    <WithStyles(Button)
      color="secondary"
      onClick={[Function]}
      variant="contained"
    >
      Registrate
    </WithStyles(Button)>
  </WithStyles(Card)>
</div>
`;
